stdlib/2.7/calendar.pyi

```diff
@@ -0,0 +1,83 @@
+from typing import Any, Iterable, Optional, Tuple
+import datetime
+
+YearType = int
+MonthType = int
+DayType = int
+WeekdayType = int
```

Comment:
But I believe that they provide useful information about what the module's functions do, though. For example, by looking at the signature of a method like `itermonthdays2` one can immediately tell from the return type that the tuples are made of a day and a weekday (and not the other way around) without having to read the docs.